lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -1,8 +1,8 @@
 param([String]$username, [String]$password, [String]$encoded_command, [String]$timeout)
 
 $task_name = "WinRM_Elevated_Shell"
-$out_file = "$env:Temp\winrm_elevated_out.log"
-$err_file = "$env:Temp\winrm_elevated_err.log"
+$out_file = "$env:Temp\winrm_elevated_out-" + "$pid" + ".log"
+$err_file = "$env:Temp\winrm_elevated_err-" + "$pid" + ".log"
```

Comment:
I'm fine with that as well if it allows us to do away with the file removal at the beginning.  I'd still want the removal at the end added under normal situations however.